### ISSUE TITLE
Update from 2018.02.2 to 2018.02.5 fails

### ISSUE THREAD

**Issue by knob-creek** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#81]
My update procedure is:
1. Shut down Apache
2. mv the old tine installation out of the way
3. unpack the new tine20-allinone
4. cp config.inc.php to the new location
5. fix ownership and permisssions
6. restart Apache
7. open setup.php, login as admin
8. update applications

This usually works fine, however, the last step failed when updating to 2018.02.5 aka 12.0
The reason is, that update_23() in Tinebase/Setup/Update/Release11.php does not seem to be called and the missing application_states tables lets other updates (Calendar, Contacts, Felanimail) fail

**Comment by ppridt** [comment]
Same problem here.

**Comment by daymien** [comment]
Me too

**Comment by gio32** [comment]
Same here. Dirty local fix (my family needs their contacts and calendar...) by
- renaming Tinebase_Setup_Update_Release12 to Tinebase_Setup_Update_Release11
- calling update_23 manually in Controller.php (similar to _fixTinebase10_33())

```
   protected function _fixTinebaseWhatever()
   {
        $updater = new Tinebase_Setup_Update_Release11($this->_backend);
        $updater->update_23();
   }

```

       $this->_fixTinebase10_33();
       $this->_fixTinebaseWhatever();

Then call

`   php /usr/share/tine20/setup.php -c /etc/tine20/config.inc.php --update
`

Updates all applications successfully now that state tables exist.
Then took back all changes in php files.

**Comment by pschuele** [comment]
hey there,
you need the following two fixes to make the update work: https://github.com/tine20/Tine-2.0-Open-Source-Groupware-and-CRM/commit/8b367aed9d2109ba7f8b6b055650d80ec7705326
https://github.com/tine20/Tine-2.0-Open-Source-Groupware-and-CRM/commit/76b988caadbfab4c539ff66eb611c64771ad6c18
we think about releasing a fixed 2018.02.6 version this thursday and are sorry for the inconvenience!

**Comment by pschuele** [comment]
duplicate of #77

**Comment by pschuele** [comment]
fixed - see #77
